src/main/kotlin/uk/gov/gdx/datashare/service/DeathNotificationService.kt

```diff
@@ -98,16 +104,28 @@ class DeathNotificationService(
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @Schema(description = "Death notification")
 data class DeathNotificationDetails(
-  @Schema(description = "First name", required = false, example = "Bob")
-  val firstName: String? = null,
-  @Schema(description = "Last name", required = false, example = "Smith")
+  @Schema(description = "Date the death was registered", required = true, example = "2022-01-05", type = "date")
+  val registrationDate: LocalDate? = null,
+  @Schema(description = "Forenames of the deceased", required = true, example = "Bob Burt")
+  val firstNames: String? = null,
+  @Schema(description = "Surname of the deceased", required = true, example = "Smith")
   val lastName: String? = null,
-  @Schema(description = "Date of Birth", required = false, example = "2001-12-31T12:34:56")
-  val dateOfBirth: LocalDate? = null,
-  @Schema(description = "Date of Death", required = false, example = "2021-12-31T12:34:56")
+  @Schema(description = "Sex of the deceased", required = true, example = "Female", allowableValues = [ "Male", "Female", "Indeterminate"])
```

Comment:
Not really in scope, but I'd be keen for this to be an enum

Reply:
Agree